vividus-plugin-aws-s3/src/main/java/org/vividus/aws/s3/steps/S3BucketSteps.java

```diff
@@ -128,6 +133,9 @@ public void uploadResource(String resourcePath, String objectKey, String content
      * @param bucketName S3 bucket to upload
      * @throws IOException in case of error on file reading
      */
+    @Deprecated(since = "0.6.0", forRemoval = true)
+    @Replacement(versionToRemoveStep = "0.7.0", replacementFormatPattern = "When I upload data `#{loadResource(%1$s)}`"
```

Comment:
how to process files?

Reply:
this step should be kept, we do not have the ability to load file content easily